### PR TITLE
Added a warning if the webadmin is enabled without any users.

### PR DESCRIPTION
**Pull request by NiLSPACE**

### CODE REVIEW

**Review by 12xx12 — APPROVED** [review]

File: src/WebAdmin.cpp [hunk 1]
```diff
@@ -190,6 +190,13 @@ void cWebAdmin::Reload(void)
 			" This will not take effect until you restart the server."
 		);
 	}
+	else if (!HasUsers())
+	{
+		LOGWARNING(
+			"The webadmin is enabled but has no users configured."
+			" To add new users see webadmin.ini"
```
Comment (madmaxoft) [inline src/WebAdmin.cpp:197]:
```suggestion
			" To add new users, edit webadmin.ini"
```

File: src/WebAdmin.cpp [hunk 2]
```diff
@@ -248,6 +255,23 @@ bool cWebAdmin::LoadIniFile(void)
 
 
 
+bool cWebAdmin::HasUsers(void)
```
Comment (madmaxoft) [inline src/WebAdmin.cpp:258]:
```suggestion
bool cWebAdmin::HasUsers()
```
`(void)` is C, `()` is C++ :)

**Review by madmaxoft — CHANGES_REQUESTED** [review]

File: src/WebAdmin.cpp (hunk 2 above)
Comment (NiLSPACE) [reply src/WebAdmin.cpp]: There are a few places where `(void)` is still used. Could be another `effort/easy` issue.

**Review by madmaxoft — APPROVED** [review]

File: src/WebAdmin.cpp (hunk 2 above)
Comment (madmaxoft) [reply src/WebAdmin.cpp]: No, the point of such easy tasks is not to offload simple mechanical work to "interns", but instead to motivate people to join. This proposition is not motivational.

**Pull request merged**